sp800-63b/sec5_authenticators.md

```diff
@@ -102,11 +104,11 @@ The out-of-band device SHALL be uniquely addressable and communication over the
 
 The out-of-band authenticator SHALL uniquely authenticate itself in one of the following ways in communicating with the verifier:
 
-- Establish an authenticated protected channel to the verifier using approved cryptography. The key used SHALL be stored in the most secure storage available on the device (e.g., keychain storage, trusted platform module, or trusted execution environment if available).
+- Establish an authenticated protected channel to the verifier using approved cryptography. The key used SHALL be stored in the most secure storage available on the device (e.g., keychain storage, trusted platform module, trusted execution).
```

Comment:
I'm not sure; I'll put it back. @garciamike please speak up if we're missing the point of your edit.